src/authorizations/components/CustomApiTokenOverlay.tsx

```diff
@@ -77,25 +77,34 @@ const CustomApiTokenOverlay: FC<Props> = props => {
   const [searchTerm, setSearchTerm] = useState('')
   const [status, setStatus] = useState(ComponentStatus.Disabled)
 
+  const {
+    allResources,
+    bucketPermissions,
+    getBuckets,
+    getTelegrafs,
+    remoteDataState,
+    telegrafPermissions,
+  } = props
+
   useEffect(() => {
-    props.getBuckets()
-    props.getTelegrafs()
-  }, [])
+    getBuckets()
+    getTelegrafs()
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
 
```

Comment:
I don't think we need to disable the warning here - is the dependency it's tracking `getBuckets` and `getTelegrafs`? Those shouldn't change, so I think they are probably safe to include in the array.

Reply:
You are correct, those two are actions and shouldn't change. Reason I left them out was to show the effect doesn't need to re-run as it doesn't depend on any values. But since adding them to the array doesn't change the logic, I'll remove the eslint comment.